lib/workers/repository/process/lookup/rollback.ts

```diff
@@ -38,7 +38,19 @@ export function getRollbackUpdate(
     'Versions found before rolling back'
   );
   lessThanVersions.sort((a, b) => version.sortVersions(a.version, b.version));
-  const newVersion = lessThanVersions.pop()?.version;
+  let newVersion;
+  const stableVersions = lessThanVersions.filter((v) =>
+    version.isStable(v.version)
+  );
+  if (version.isStable(currentValue)) {
+    newVersion = stableVersions.pop()?.version;
+  }
+  if (!version.isStable(currentValue)) {
+    newVersion = lessThanVersions.pop()?.version;
+  }
+  if (!newVersion) {
```

Comment:
The use of this if and the if above it makes no sense because they have the same code inside